apps/web/src/state/app/authentication.ts

```diff
@@ -188,16 +198,121 @@ export const storeAuthenticatedRecords = async (
 	login?: UserCredential,
 ): Promise<void> => {
 	await key.reconstructKey();
-	const privateKeys = await key.modules.privateKeyModule.getPrivateKeys();
 
 	if (login?.user) {
 		await setProfile(makeProfile(login));
 	}
 
+	// await initByPrivateKeyModule(passcode);
+	await initBySeedPhraseModule(passcode);
+	await key.syncLocalMetadataTransitions();
+
+	modules.engine.start();
+};
+
+const defaultPrefixDerivationPaths: Array<{ path: string; network: Networks }> =
+	[
+		{
+			path: "44'/501'",
+			network: Networks.solana,
+		},
+		{
+			path: "44'/784'",
+			network: Networks.sui,
+		},
+		{
+			path: "44'/1729'",
+			network: Networks.tezos,
+		},
```

Comment:
maybe a comment which explain where these `path` come from and/or how to find them in docs would be appreciated

Reply:
these paths are default paths for each network, we can find them in the dev docs of each network. I also noted this in [notion task](https://www.notion.so/stormgate/2a39a522cbd443a59ee2eb06ebb9dba2?v=1904b777f28648f28d21db626f9bfec4&p=83aad874791445d880937c1bee6a41ea&pm=s)